pkg/builder/builder_suite_test.go

```diff
@@ -63,7 +65,7 @@ var _ = BeforeSuite(func() {
 
 	webhook.DefaultPort, _, err = addr.Suggest("")
 	Expect(err).NotTo(HaveOccurred())
-}, 60)
```

Comment:
Why is the custom timeout gone?

Reply:
This has been removed in v2.

Reply:
I checked, but I guess there is no support. Please check https://onsi.github.io/ginkgo/MIGRATING_TO_V2#major-additions-and-improvements.

Reply:
Please describe in your PR how the timeout behavior for these kind of tests was before and how it is now. Just removing things without a proper explanation is not the way to go.

Reply:
I have updated the PR and resolved your concern, and also opened an issue on gingko [here](https://github.com/onsi/ginkgo/issues/940). PTAL

Reply:
Previously custom timeouts were there to run tests asynchronously, now ginkgo doesn't support async testing. I have updated it in the PR description.

Reply:
@alvaroaleman in ginkgo v1 `BeforeSuite` honored the timeout if and only if the function to `BeforeSuite` accepted a `done` channel - which is not the case with the code before this change. So even before this issue the timeout was ignored, so removing it here is fine.